powerline.el

```diff
@@ -467,9 +468,13 @@ static char * %s[] = {
    "\\`[ \t\n\r]+" ""
    (replace-regexp-in-string "[ \t\n\r]+\\'" "" s)))
 
+(defun strip-text-properties (txt)
```

Comment:
I still prefer to keep it a separate function because it makes it easier to read, avoid repeating `(powerline-trim (format-mode-line mode-line-buffer-identification))` in the code and it makes it reusable for later (let's say if one wants to strip face properties of `which-func-format` for instance).

I will call it `powerline-strip-text-properties` if that is ok for you (or do you really want the two dashes ?).

Btw, i saw two styles of namespacing in your code, is there any particular meaning to `pl/` or `powerline-` prefixes ?